spec/rubocop/cop/factory_bot/consistent_parentheses_style_spec.rb

```diff
@@ -405,6 +405,15 @@
         generate(:foo, :bar)
       RUBY
     end
+
+    context 'when TargetRubyVersion >= 3.1', :ruby31 do
+      it 'does not register an offense when using `create` ' \
+         'with pinned hash argument' do
+        expect_no_offenses(<<~RUBY)
+          create(:user, name:)
```

Comment:
Will we also ignore when there’s more than one like `name:, foo:`?

Reply:
Yes. I re-pushed one change that I forgot to push.